components/esp_peripherals/periph_wifi.c

```diff
@@ -87,7 +87,7 @@ esp_err_t periph_wifi_wait_for_connected(esp_periph_handle_t periph, TickType_t
     if (connected_bit & CONNECTED_BIT) {
         return ESP_OK;
     }
-#if defined(CONFIG_BTDM_CTRL_MODE_BLE_ONLY) || defined(CONFIG_BTDM_CTRL_MODE_BTDM)
+#if defined(CONFIG_BT_BLE_BLUFI_ENABLE)
     if (periph_wifi->config_mode == WIFI_CONFIG_BLUEFI) {
         ble_config_stop();
```

Comment:
`ble_config_start` in `periph_wifi_config_start` also need be changed

Reply:
Good catch, I wonder why that never gave me any compile issues. 🤔